Formula/openjdk.rb

```diff
@@ -5,6 +5,11 @@ class Openjdk < Formula
   sha256 "9c5be662f5b166b5c82c27de29b71f867cff3ff4570f4c8fa646490c4529135a"
   license :cannot_represent
 
+  livecheck do
+    url "https://hg.openjdk.java.net/jdk-updates/jdk15u/tags"
```

Comment:
```suggestion
    url "https://hg.openjdk.java.net/jdk-updates/jdk#{version.major}u/tags"
```
Can you see if this will work?

Reply:
It works on my machine

Reply:
I'll push the changes